### PR TITLE
API

### PR DESCRIPTION
**Pull request by i-max-xi**
API connected ✔️

### CODE REVIEW

File: src/Components/Book.js [hunk 1]
```diff
@@ -1,15 +1,24 @@
...
+//       <h1>{props.renderedBook.title}</h1> 
+//       <h3>{props.renderedBook.author}</h3>
+//       <button type='button'>Comments</button>
+//       <button type='button' onClick={onSaveDelete}>Remove</button>
+//       <button type='button'>Edit</button>
+//     </React.Fragment>
+//   );
+// };
+
+// export default Book;
```
Comment (smunozmo) [inline src/Components/Book.js]: - [ ] You are almost there! It would be great to remove any unused code to keep your code clean.

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @i-max-xi, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!


## Required Changes ♻️

- [ ] Your app is looking great, well done! However, I was not able to test some features because there was a problem when I tried to add a new book:

[The Bookstore app - Video](https://www.loom.com/share/5dff5dd924b1478a9d3f723ea8d42fa1)

It would be great to fix this problem, let me know if this is happening only on my machine.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Book.js (hunk 1 above)
Comment (i-max-xi) [reply src/Components/Book.js]: Thank you
I have noticed the issue. I am fixing it now

Comment (rloterh) [reply src/Components/Book.js]: This issue is well resolved! ✔️

Comment (i-max-xi) [reply src/Components/Book.js]: Thank you,
I would be waiting for the approved message 😉

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Max,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- API fetch properly implemented
- State management properly implemented

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comments marked **[OPTIONAL]** under this review_

You can also consider:
- [OPTIONAL] Kindly ensure to provide a descriptive PR message. This will give a good overview of your project to the code reviewer.

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/Book.js (hunk 1 above)
Comment (smunozmo) [reply src/Components/Book.js]: Nice job! 😎

**Pull request merged**